python/tests/test_options.py

```diff
@@ -0,0 +1,80 @@
+from simplesauce.options import SauceOptions
+
+
+class TestInit(object):
+
+    def test_defaults_to_win10_chrome_latest(self):
+        sauce = SauceOptions()
+
+        assert sauce.browserName == 'chrome'
+        assert sauce.browserVersion == 'latest'
+        assert sauce.platformName == 'windows 10'
+
+    def test_accepts_browser_version_platform_name(self):
+        sauce = SauceOptions('Firefox', '67.0', 'windows 10')
+
+        assert sauce.browserName == 'Firefox'
+        assert sauce.browserVersion == '67.0'
```

Comment:
We need to figure out across the languages how we're going to be able to support the valid browser versions outside of pushing changes to these repos every time it changes. I wonder if we could create a dead-simple call to the Platform Configurator to create a cache of valid browser versions, and create defaults for them. This is a larger discussion. Ideally it would live in a "common" area of the project, usable by all languages...

Reply:
This is probably the right way to go. In this case, I'm naively validating that what version is passed in is kept true by the `SauceOptions` instance. Integrating the Platform Configurator somehow to get valid/invalid/up to date versions of browsers would be excellent, and could definitely live in a common area.